mitosheet/src/components/elements/TextButton.tsx

```diff
@@ -70,6 +70,11 @@ interface TextButtonProps {
         * @param [disabledTooltip] -- Message to display as tooltip when button is disabled 
     */
     disabledTooltip?: string
+
+    /**
+     * Boolean to indicate whether the button is showing error. Impacts color. 
```

Comment:
This doesn't appear to impact color any more than disabled. Can we rip this prop out and just pass things through disabled for now?